src/components/IPFS/Dropzone.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import { ReactComponent as UploadIcon } from '../Icons/Upload.svg'
+
+const DropzoneArea = styled('div')`
+  height: 200px;
+  width: 200px;
+  background-color: #fff;
+  border: 2px dashed rgb(187, 186, 186);
+  border-radius: 2%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  font-size: 16px;
+`
+
+const Icon = styled(UploadIcon)`
+  opacity: 0.3;
+  height: 64px;
+  width: 64px;
+`
+
+const IconText = styled('span')`
+  opacity: 0.3;
+`
+
+const Dropzone = props => {
+  const [highlight, setHighlight] = useState(false)
```

Comment:
`highlight` does not seem to be used

Reply:
Got rid of this when switching to the two button approach mentioned below